2.10/2.10.15/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.10.15
+Выводится сообщение с кнопками ОК и ОТМЕНА: "На вашей карточке Х руб. Хотите снять?"
+Где Х - 100.
+Если юзер выбирает отмену - вывести сообщение "Заберите карточку".
+Если юзер выбирает ОК - спросить у него: "Какую сумму?"
+Если сумма будет от 0 по 100 - тогда вывести сообщение "На вашей карточке осталось Y руб."
+Где Y - оставшееся сумма.
+Если юзер введет неправильную сумму - тогда вывести сообщение "Сумма неверная, попробуйте еще раз."
+<script>
+	let haveMoney = 100;
+	let withdrawMoney;
```

Comment:
when we start name from a verb it means that a variable is a function which will do some action.
Once again:
1) if a variable has boolean type - its name should start with "is, has, have", for example: **isBigNumber**, **hasError** etc
2) all other variables should be a noun, for example: **smallMessage**, **errorQuantity** (or errorNumbers), **grandTotal**, **resultSumma** etc